R/qc_fix.R

```diff
@@ -65,9 +65,13 @@ create_comment_body <- function(owner,
                                 comparator_commit = "current") {
 
   issue <- get_issue(owner, repo, issue_number)
+  ## check if file exists locally
+  if (!fs::file_exists(issue$title)) {
+    rlang::abort(glue::glue("Error: {issue$title} does not exist in local project repo"))
+  }
```

Comment:
Yikes this is sorta bad - I realized in implementing feedback that there's no error handling for when a file with an associated ghqc issue doesn't exist locally 

<img width="853" alt="Screenshot 2024-10-24 at 1 34 45 PM" src="https://github.com/user-attachments/assets/ba14783d-c519-43d5-b486-6c670575421a">